src/scanners/css.js

```diff
@@ -26,7 +26,15 @@ export default class CSSScanner extends BaseScanner {
                 }
 
                 this.validatorMessages = this.validatorMessages.concat(
-                  rules[cssRule](rule));
+                  rules[cssRule](
+                    rule.declarations,
```

Comment:
> Isn't this what the {}={} is for in the rule function?

Could do but it seems better to provide the entire cssRule object to all validator rule functions than selectively provide specific data for a subset of the rules.  To clarify I'm only talking about the first arg. Passing other more specific bits of data via other args is fine - especially along the lines of making the API more generic. 

You could use a destructuring assignment [1] to pull out the other bits that you're explicitly passing - but it might be simplest to send the whole object verbatim despite there being some duplication in what you send as other args.

[1] http://jsbin.com/rohegudeno/edit?js,console